scripts/public_symbols_checker.py

```diff
@@ -81,11 +84,12 @@ def m_diff_lines_getter(diff_lines):
 
 get_symbols("A", a_diff_lines_getter, r"")
 get_symbols("M", m_diff_lines_getter, r"\+")
+get_symbols("M", m_diff_lines_getter, r"\-", remove=True)
 
-if file_path_symbols:
+if symbols_changed:
```

Comment:
Could you just check `if added_symbols or removed_symbols: ...` and remove this variable or is it different?

Reply:
It is not, will update it